### PR TITLE
✨ Add LobbyListItem component

### PR DESCRIPTION
**Pull request by fre-ben**
- Add LobbyListItem component
- Add LobbyListItem to storybook
- Adjust style of PlayerCount component
- Add disabled button state
- Add playerCount monitoring to LobbyListItem component

Deployment: https://sisu-pipelin-lobbyitem-7gsjjqb.herokuapp.com/storybook/index.html?path=/story/common-misc--lobbylistitem
Part of Issue #1

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! 🚀 Nice subtle hover effect

File: components/button/Button.module.css [hunk 1]
```diff
@@ -14,6 +14,15 @@
...
 }
 
+.btn:disabled {
+  background-image: linear-gradient(
+      var(--placeholder-color),
+      var(--placeholder-color)
+    ),
+    linear-gradient(var(--placeholder-color), var(--placeholder-color));
+  color: var(--placeholder-color);
+}
```
Comment (mhstrkmp) [inline components/button/Button.module.css:24]: I don't understand this. If you have a linear-gradient with the same colors it isn't a gradient at all.

**Review by mhstrkmp — COMMENTED** [review]
lgtm 👍

File: components/button/Button.module.css (hunk 1 above)
Comment (fre-ben) [reply components/button/Button.module.css:24]: Yeah, that's a bit of CSS magic.
Doens't work if it isn't written like this.
See this source: https://stackoverflow.com/questions/4131490/any-way-to-limit-border-length/42157360#42157360

**Pull request merged**